app/views/employees/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="main-container">
+  <div class="register-container">
+    <section class="title-container">
+      <h1>Employee edit</h1>
+    </section>
+    <section class="form-container">
+      <%= form_with model: @employee do |form| %>
+        <div class="info-container">
+          <%= form.label :email %><br>
+          <%= form.text_field :email %>
+        </div>
+        <div class="info-container">
+          <%= form.label :name %><br>
+          <%= form.text_field :name %>
+        </div>
+        <div class="info-container">
+          <%= form.label :position %><br>
+          <%= form.text_field :position %>
+        </div>
+        <div class="info-container">
+          <%= form.label :employee_number %><br>
+          <%= form.text_field :employee_number %>
+        </div>
+        <div class="info-container">
+          <%= form.label :private_number %><br>
+          <%= form.text_field :private_number %>
+        </div>
+        <div class="button-container">
+          <%= form.submit %>
+        </div>
+      <% end %>
+    </section>
+  </div>
+</div>
```

Comment:
Add a blank line at the end 
Also aren't edit and new forms the same what do you think about using [partial](https://guides.rubyonrails.org/getting_started.html#rendering-a-partial-form)